optimum/exporters/openvino/convert.py

```diff
@@ -23,6 +23,7 @@
 from transformers.generation import GenerationMixin
 from transformers.utils import is_tf_available, is_torch_available
 
+from openvino.frontend.pytorch.ts_decoder import TorchScriptPythonDecoder
```

Comment:
I would prefer if we used something like `TorchScriptPythonWrapper` or `TorchScriptPythonModel`, the decoder part doesn't make sense especially since we are using this wrapper on top of diffusers text_encoder/unet/transformer etc.
Will this wrapper ever change and do decoder-specific stuff ? if npt why does it have `Decoder` in its name.

Reply:
This "decoder" in a sense that it is used to decode the pytorch model. It is used in openvino as an abstraction to get the graph from the model. It is an internal class, but it can be used to provide such custom parameters for tracing.

Reply:
this naming does not have relation to decoder models, Decoder is internal component of openvino responsible for decoding of original framework model during conversion. All that time we use it internally inside convert_model, now it is created explicitly for resolving advanced conversion options

Reply:
oh okay, confusing name but okay if it's part of the openvino naming conventions.

Reply:
Can we merge this then?